build_tools/install_rocm_from_artifacts.py

```diff
@@ -204,22 +196,15 @@ def retrieve_artifacts_by_release(args):
         release_tag = "nightly-release"
     # Otherwise, determine if version is nightly-release or dev-release
     else:
-        try:
-            version = Version(args.release)
-            if not version.is_devrelease and not version.is_prerelease:
-                log(f"This script requires a nightly-release or dev-release version.")
-                log("Please retrieve the correct release version from:")
-                log(
-                    "\t - https://github.com/ROCm/TheRock/releases/tag/nightly-release (nightly-release example: 6.4.0rc20250416)"
-                )
-                log(
-                    "\t - https://github.com/ROCm/TheRock/releases/tag/dev-release (dev-release example: 6.4.0.dev0+8f6cdfc0d95845f4ca5a46de59d58894972a29a9)"
-                )
-                log("Exiting...")
-                return
-            release_tag = "dev-release" if version.is_devrelease else "nightly-release"
-        except InvalidVersion:
-            log(f"Invalid release version '{args.release}'")
+        # Searching for nightly-release or dev-release format
+        nightly_regex_expression = (
+            "(\\d+\\.)?(\\d+\\.)?(\\*|\\d+)rc(\\d{4})(\\d{2})(\\d{2})"
+        )
+        dev_regex_expression = "(\\d+\\.)?(\\d+\\.)?(\\*|\\d+).dev0+"
+        nightly_release = re.search(nightly_regex_expression, args.release) != None
+        dev_release = re.search(dev_regex_expression, args.release) != None
+        if not nightly_release and not dev_release:
+            log(f"This script requires a nightly-release or dev-release version.")
             log("Please retrieve the correct release version from:")
```

Comment:
nit: drop `f` if not using any substitutions (mostly for consistency with the following line)
```suggestion
            log("This script requires a nightly-release or dev-release version.")
            log("Please retrieve the correct release version from:")
```